src/runtime/arm_cpu_features.cpp

```diff
@@ -22,6 +22,8 @@ WEAK CpuFeatures halide_get_cpu_features() {
 
     // TODO: add runtime detection for ARMDotProd extension
     // https://github.com/halide/Halide/issues/4727
+
+    // TODO: add runtime detection for ARMFp16 extension
```

Comment:
Should probably open a tracking issue for this, too

Reply:
Filed #6106